internal/pkg/service/appsproxy/dataapps/api/notify.go

```diff
@@ -4,18 +4,20 @@ import (
 	"time"
 
 	"github.com/keboola/go-client/pkg/request"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/service/common/utctime"
 )
 
 type notifyBody struct {
-	LastRequestTimestamp string `json:"lastRequestTimestamp"`
+	LastRequestTimestamp utctime.UTCTime `json:"lastRequestTimestamp"`
```

Comment:
We have already type for it, plus UTC timezone is better for debuggng, and it doesn't depend of the server timezone settings.